compiler/luci/log/src/Log.cpp

```diff
@@ -65,6 +70,10 @@ LoggerConfig::LoggerConfig()
 
   // Turn on info logging if LUCI_LOG is set as non-zero value
   _show_info = safecast<bool>(std::getenv("LUCI_LOG"), false);
+
+  // Turn on verbose logging if LUCI_LOG is set to some level
+  // VERBOSE(l, 1) will be visible with LUCI_LOG=2 and VERBOSE(l, 2) with LUCI_LOG=3 and so on
```

Comment:
I'm just confused with the number. Isn't it `VERBOSE(l, 1) will be visible with LUCI_LOG=1`?

Reply:
`VERBOSE(l, level)` will be visible for higher than the `level`, that I've discovered while running with this. that is `VERBOSE(l, 0)` is same(?) as `INFO(l)`